src/project-2.js

```diff
@@ -36,9 +62,23 @@ const fizzBuzz = (num) => {
   // if num is divisible by 5 return 'buzz'
   // if num is divisible by 3 & 5 return 'fizzbuzz'
   // otherwise return num
+  if (num % 3 === 0 && num % 5 === 0) {
+    return 'fizzbuzz';
+  } else if (num % 5 === 0) {
+    return 'buzz';
+  } else if (num % 3 === 0) {
+    return 'fizz';
+  }
+  return num;
 };
 
 const isPrime = (num) => {
```

Comment:
- You don't need `num.length` here because you want the for loop to go until `num`.
- You are returning true too early.
- You are returning false at the end of the for loop, but if num is not divisible by any number under it, then it is prime. Otherwise, it is not prime.
- Make sure to try testing your code in a REPL to see if your cases are covered.